src/components/TodoList/Item/TodoItem.module.scss

```diff
@@ -9,12 +9,25 @@
   border: 1px solid $clr_orange_primary;
 }
 
+.main {
+  display: flex;
+  gap: 10px;
+}
+
 .text {
   padding-bottom: 10px;
 
   overflow: auto;
 }
 
+.text_crossed {
+  padding-bottom: 10px;
+
+  overflow: auto;
+
+  text-decoration: line-through;
+}
+
```

Comment:
Better to have text_crossed as additional class and not duplicate all styles for text

Reply:
Separated text_crossed class into two different classes -> text and crossed